roles/accprovision/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# defaults file for AccProvision
+acc_provision_dir_path: /tmp/acc_provision
+acc_provision_file_name: acc_provision_input.yaml
+acicnioperator_cr: cr.yaml
```

Comment:
Nomenclature cr.yaml might not be accurate

Reply:
We have now moved to creating a deployment file instead and called the variable aci_cni_deployment_file